README.md

```diff
@@ -385,6 +385,7 @@ You can reach me on [Twitter @ileriayooo](https://twitter.com/Ileriayooo)
/NbxEuWiE1xY8NCLUIG9FgNKnhgIiLgFswSXabGx95PvfUWaVsAMXdgXwNyIxKoBCrUAjvLL9XxCJYxEglEu/oRw0wis7ApixgC2FTQ2p4dX5vy/+MAXw0EfeKJwkgB6/YFTC1ND6CmDUcvzJAuirhCVYgXUBREsAKwScBdxuuLAFNcoKlVm4F8x3ADXg4GcoF4jIOTlA5YA+AWzAI/pyfBXAPFpwie7/VQDzYPu1IDJVAPP4DC7VAjIj2wJQoApgIgotYH+QgCqA6TTgIkhAFcB0WnBJ0pFdBTAd6hJwjBbwIaoAptOChyARVQATUKjXR2kBH6IKYBrZF4ACVQDTyL4AFKgCmAYzAnR9r0USqgDORKEFLJn/e6oAzqcY//e8gxK+U8ZdviA97GNWsu4BPEQzAlhpImWHWGoCmKoAFFATgKGjZwRcGvAQJEY1B3AiuHOXO6RFQKK0BNATIwlMaQXsbWBFJYAedQEktgLrh0B0SEyUaWBCK2A3WTITwGhnAIwRsw6QwgoEXKgtYDBANAEksgLaTVZoATOxBSxqJTCyFbCbLBtwERggRSk4lhWwR1gLLsuygEBEKxBwYbaAJWkBHyLJYlAkK7C8DdzE6PekXA3UtALqGnupCaAnmQCUrYBdYi1qBXCXpP0AilYg4FJcAShgoSFEwwoEXIr0f09yAShYgcYIYwrA1BlAJlrCyFZAffEVjoEVGMJSTyDLCtgjrNgE0GNGAEQrEHBhJoBmCkABU13BJCtgj7AGPJKcATCGxbbwOVZAHWEKLWACY5gTwEwrqP5/JiY3hsywAgGXFkRSt4APYXln0BQrsPwcAIFBzApgghVobLJswSN5C/gQpvcGnmkF1BvcPwmcuQIoMEgOm0NPtQIBl+ITQI/a5lAW3grcaPQiuD3yowIuLbg8uf8Dxuiy2B5+ghXk8CAoizzmdD7AmBXk8CAoi9xlI4AjswL2g6DYLWBWecjqhJARKxBwWcLo39pmjkfEDFmB5QYQq2zvWXYCGLACjSXWJSSA9/5dlodEHViBRotVi/LJMwLsEKxAQEShBcwkYWEqWwHsWEH1//OR8IH5SuAYvRWwYR8Da5GXdZN6UORrGpTPy8CpTw7doW8B+4vyuQgzpxoB9lnE/H932lwFsM9iCkCBKoB9lpAA3u9+UgWwz+IiwD/gJHOrDPZxTAAAAABJRU5ErkJggg==)                        | `![Apache Lucene](https://img.shields.io/badge/-Apache%20Lucene-019B8F?style=for-the-badge&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAIAAAABRCAYAAAAaXK5BAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAARkSURBVHgB7Z2NUdwwEIXf3aSA0IFTQdJBXEKoIEMHpIOkg6QDOgA62HQAFeBUEDq4SBeLuTuM785+K61kfTNg/mYA39O+1WolrzabzQ2Ar1gmV6vV6gYk3L1s3OUJPDr3932AImv3du1/EZbJA7h8ApdHKLN2Cnt21yssEPe/swXQgotAGR8B/I0Qd/mFZSHg8xFc2AJ9xTp84ERwHeMXGkIjvFItoB+YqqwPPr90b89YBgIiLgH0L/578BBEYE8ATnGdu/zAMujApQEX9QTQcxgBvAh+IpL6EvJcE8D/rN/4eulWoJHrsBPADhEYFEA/NbxEuWiE1xY8NCLUIG9FgNKnhgIiLgFswSXabGx95PvfUWaVsAMXdgXwNyIxKoBCrUAjvLL9XxCJYxEglEu/oRw0wis7ApixgC2FTQ2p4dX5vy/+MAXw0EfeKJwkgB6/YFTC1ND6CmDUcvzJAuirhCVYgXUBREsAKwScBdxuuLAFNcoKlVm4F8x3ADXg4GcoF4jIOTlA5YA+AWzAI/pyfBXAPFpwie7/VQDzYPu1IDJVAPP4DC7VAjIj2wJQoApgIgotYH+QgCqA6TTgIkhAFcB0WnBJ0pFdBTAd6hJwjBbwIaoAptOChyARVQATUKjXR2kBH6IKYBrZF4ACVQDTyL4AFKgCmAYzAnR9r0USqgDORKEFLJn/e6oAzqcY//e8gxK+U8ZdviA97GNWsu4BPEQzAlhpImWHWGoCmKoAFFATgKGjZwRcGvAQJEY1B3AiuHOXO6RFQKK0BNATIwlMaQXsbWBFJYAedQEktgLrh0B0SEyUaWBCK2A3WTITwGhnAIwRsw6QwgoEXKgtYDBANAEksgLaTVZoATOxBSxqJTCyFbCbLBtwERggRSk4lhWwR1gLLsuygEBEKxBwYbaAJWkBHyLJYlAkK7C8DdzE6PekXA3UtALqGnupCaAnmQCUrYBdYi1qBXCXpP0AilYg4FJcAShgoSFEwwoEXIr0f09yAShYgcYIYwrA1BlAJlrCyFZAffEVjoEVGMJSTyDLCtgjrNgE0GNGAEQrEHBhJoBmCkABU13BJCtgj7AGPJKcATCGxbbwOVZAHWEKLWACY5gTwEwrqP5/JiY3hsywAgGXFkRSt4APYXln0BQrsPwcAIFBzApgghVobLJswSN5C/gQpvcGnmkF1BvcPwmcuQIoMEgOm0NPtQIBl+ITQI/a5lAW3grcaPQiuD3yowIuLbg8uf8Dxuiy2B5+ghXk8CAoizzmdD7AmBXk8CAoi9xlI4AjswL2g6DYLWBWecjqhJARKxBwWcLo39pmjkfEDFmB5QYQq2zvWXYCGLACjSXWJSSA9/5dlodEHViBRotVi/LJMwLsEKxAQEShBcwkYWEqWwHsWEH1//OR8IH5SuAYvRWwYR8Da5GXdZN6UORrGpTPy8CpTw7doW8B+4vyuQgzpxoB9lnE/H932lwFsM9iCkCBKoB9lpAA3u9+UgWwz+IiwD/gJHOrDPZxTAAAAABJRU5ErkJggg==)`
```

Comment:
Cumbersome badge metadata.
KIS - Keep it simple.

Reply:
I am just waiting the approval of the Apache Lucene icon on simple-icons.

That base64 was just a POC